### DISCUSSION TITLE
768

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#937] [Comments]
# 768

요즘 좋은 기회로 여러 리더분들을 뵙고 있다. 한달에 1~2번정도 CEO, CTO, CPO 분들을 뵙고 서로 궁금한 것들을 물어보고 이야기를 나누는 시간을 가지고 있다. 이 분들과 대화를 나누다보면 "오? 그러네요?" 라는 감탄사를 낼 때가 자주 있다. 그러면서 어떻게 이런 제품 아이디어를 가지셨을까? 혹은 이런 조직 관리 쳬계를 가지시게 된 걸까 궁금해서 사적인 이야기들로 이어진다. 그런 이야기들을 정리해보면 공통적으로 "인간의 본성은 어떠하다" 라는 본인만의 주관적인 정의가 내려진 분들이 좋은 제품 혹은 좋은 제품을 만드는 것 같았다. "아! 사람이란 이런 존재구나" 라는 것의 기준이 있는 사람들은 그 기준에 맞춰서 조직 관리나 제품의 방향성을 결정하는것 같은 느낌을 받았던 것이다. 데이터를 통해 결..

https://jojoldu.tistory.com/768

**Comment by chrishoonS** [comment]
이런 것들을 생각하고 고뇌하며 글을 쓰는 리더는 현업에서는 어떠실지 참 궁금합니다! 좋은 글 잘보고가요:)

**Comment by WoongE** [comment]
> 그런 이야기들을 정리해보면 공통적으로 "인간의 본성은 어떠하다" 라는 본인만의 주관적인 정의가 내려진 분들이 좋은 제품 혹은 좋은 제품을 만드는 것 같았다.
> "아! 사람이란 이런 존재구나" 라는 것의 기준이 있는 사람들은 그 기준에 맞춰서 조직 관리나 제품의 방향성을 결정하는것 같은 느낌을 받았던 것이다.

이 부분 참 공감됩니다.
저는 저걸 '통찰'이라고 정의하고 있고,
저런 통찰을 하나씩 학습하고 쌓아나가야 어떠한 상황의 본질을 꿰뚫어보는 '통찰력'이란 초능력이 생기는 거라고 생각합니다.

참고로, <군주론>을 읽어보시면 마키아벨리의 인간에 대한 통찰을 많이 보실 수 있습니다. (제 인생책 딱 한 권만 꼽으라고 하면 무조건 이 책을 얘기할 거에요.)
또한 읽어보시고 마키아벨리의 학습/연구 방식을 생각해보시면, '역사'는 이 통찰을 학습하기 위한 도구라는 것도 배울 수 있습니다.
마키아벨리는 참고로 로마 역사에 관심이 많았고, 여기서 통찰을 많이 얻었던 걸로 보입니다. <로마사 논고>라는 책까지 썼을 정도에요.

재밌는 글 잘 읽었습니다~ :)
